engine/console/game.go

```diff
@@ -7,31 +7,47 @@ import (
 	"log"
 	"math/rand"
 	"os"
+	"strconv"
 	"time"
+
+	//"github.com/sanity-io/litter"
 )
 
 type Snake struct {
-	id    int          // snakep2p id
-	body  []core.Coord // coordinates of snakep2p's body
-	head  core.Coord   // coordinates of snakep2p's head
-	style tcell.Style  // snakep2p's style
+	alive bool         // whether snake is alive or not
+	Body  []core.Coord // coordinates of snakep2p's body
+	Head  core.Coord   // coordinates of snakep2p's head
+	Style tcell.Style  // snakep2p's style
```

Comment:
Remove comments that duplicate what the name and the type say and make `alive` public, please.